includes/class-wc-payments.php

```diff
@@ -561,6 +559,16 @@ public static function init() {
 
 		self::maybe_display_express_checkout_buttons();
 
+		// Insert the Stripe Payment Messaging Element only if there is at least one BNPL method enabled.
+		$activated_bnpl_payment_methods = array_intersect(
```

Comment:
Given the context, the naming should be `enabled_bnpl_payment_methods` - because of `howget_upe_enabled_payment_method_ids` is named.